Content.Server/GameObjects/Components/Interactable/Tools/WelderComponent.cs

```diff
@@ -159,8 +45,12 @@ void IExamine.Examine(FormattedMessage message)
                 message.AddText(loc.GetString("Not lit\n"));
             }
 
-            message.AddMarkup(loc.GetString("Fuel: [color={0}]{1}/{2}[/color].",
-                Fuel < FuelCapacity / 4f ? "darkorange" : "orange", Math.Round(Fuel), FuelCapacity));
+        }
+
+        public bool UseEntity(UseEntityEventArgs eventArgs)
```

Comment:
You don't need to implement `IUse` anymore, the `WelderSystem` handling works either way.